sys/vane/clay.hoon

```diff
@@ -2372,9 +2372,17 @@
     ++  read-p-in
       |=  {pax/path pes/regs}
       ^-  dict
-      =+  rul=(~(get by pes) pax)
-      ?^  rul  [pax u.rul]
-      ?~  pax  [/ %white ~]
+      =/  rul=(unit rule)  (~(get by pes) pax)
+      ?^  rul
+        :+  pax  mod.u.rul
+        %-  ~(rep in who.u.rul)
+        |=  {w/whom out/(pair (set @p) (map @ta crew))}
+        ?:  ?=([$& @p] w)
```

Comment:
You're mixing old and new syntax here, as well as other places in the pr. Clay in general still uses old syntax, you might want to stick to that for consistency, but since this change is fairly self-contained (in clay) I see no harm in applying new syntax for everything you touch. Just be consistent about it.